code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1356,6 +1359,10 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	..()
 	. = 1
 
+/datum/reagent/consumable/ethanol/neurotoxin/on_mob_delete(mob/living/carbon/M)
+	if(p)
+		QDEL_NULL(p)
```

Comment:
Isn't there a proc for removing traumas?

Reply:
why are you reviewing a placeholder

Reply:
Because no implying knowledge of PR progress